win32ss/user/ntuser/ime.c

```diff
@@ -1142,11 +1144,199 @@ NtUserSetImeInfoEx(PIMEINFOEX pImeInfoEx)
     return ret;
 }
 
+// Win: ImeSetFutureOwner
+VOID FASTCALL IntImeSetFutureOwner(PWND pImeWnd, PWND pwndOldOwner)
```

Comment:
Comments about the purposes of such functions and its reasons to be used in the code would be welcome.